src/status_im/events.cljs

```diff
@@ -1237,7 +1242,8 @@
 (fx/defn on-going-in-background [{:keys [db now]}]
   {:db (-> db
            (dissoc :app-active-since)
-           (assoc :app-in-background-since now))})
+           (assoc :app-in-background-since now))
+   :dispatch-n '([:audio-recorder/on-background] [:audio-message/on-background])})
```

Comment:
just wondering why list?

Reply:
@flexsurfer you're concerned because i fire two events or because it is a list?

Reply:
last one :) usually vector is used

Reply:
don't even remember from where i took that :) i'll change it later